src/components/cylc/Header.vue

```diff
@@ -2,7 +2,8 @@
   <v-list-tile-action-text class="pt-3 pb-3 c-header">
     <v-layout align-center justify-center row wrap fill-height>
       <div class="mb-2">
-        <svg version="1.1" viewBox="0 0 655 260" xmlns="http://www.w3.org/2000/svg" xmlns:osb="http://www.openswatchbook.org/uri/2009/osb">
+        <svg version="1.1" preserveAspectRatio="xMinYMin slice" width="100%" height="100%" viewBox="0 0 655 260" xmlns="http://www.w3.org/2000/svg"
```

Comment:
We want `meet` rather than `slice` as `slice` can crop the SVG

https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/preserveAspectRatio#Syntax

Reply:
Aaahh. I will admit that I blindly copied the code without checking the slice. Done! Thanks!